Automattic/wp-calypso
# Build Scripts: Add build-client-if-desktop

There are two environments that need to write js files to the filesystem during build:
1. production
2. desktop

With [recent changes](https://github.com/Automattic/wp-calypso/pull/18074) in the setup, `desktop` support was dropped.  This PR brings it back.

To Test
----

1. Go to `wp-desktop` and checkout `release/3.0.0`
2. cd into the `calypso` folder, checkout this branch (`desktop/fix-build-minify`)
2. run make build in the `wp-desktop` Makefile. you should see `bundle.js` output in the `calypso/public` folder
